client/src/pages/GamesStart.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
```

Comment:
@dropcat13 I think we decided this page would just always be combined with the "Home" page, right?